feat: document open-api lib (#174)
ts-rest/ts-rest: apps/docs/docs/guides/non-ts-clients.mdx

# Without TS Clients

> In a brownfield code base it's common that you may not have the luxury of having Typescript end to end, possibly you've got a Android/iOS/Flutter app that's written in Java/Kotlin/Swift and you're looking to add Typescript to your backend.

Reviewer: ```suggestion
In a brownfield code base, it's common that you may not have the luxury of having Typescript end-to-end. For instance, you might have an Android/iOS/Flutter app written in Java/Kotlin/Swift and want to add Typescript to your backend.
```
